src/style/components/Console.ts

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+
+export const WrapperConsole = styled.div`
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+`;
+
+export const Main = styled.main`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+`;
+
+export const SectionHistory = styled.section`
+  display: flex;
+  width: 100%;
+  height: 50px;
+  padding: 10px 15px;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.2);
+`;
+
+export const SectionQuery = styled.section`
+  display: flex;
+  background: #ffffff;
```

Comment:
If you use one color more then 2-3 times in the app, please put it into variable and just import everywhere. For example the app gonna be redesigned and it would be much easier to change main colors.

Reply:
done